lib/coverage.js

```diff
@@ -251,6 +251,9 @@ internals.instrument = function (filename) {
         loc: true,
         comment: true,
         range: true,
+        ecmaFeatures: {
+          experimentalObjectRestSpread: true
```

Comment:
need to indent this a tad more, that is why the CI is failing

Reply:
yeh before prettier died reindint just everything ;) Would be awesome if you could add an .eslintrc to your project so prettier can do that stuff for me :)

Reply:
https://github.com/hapijs/lab/blob/master/lib/linter/.eslintrc.js

Reply:
@cjihrig That should be in the project root actually.

Reply:
Nonetheless, that's where it is if you want to access it.